### ISSUE TITLE
Juz Pie charts

### ISSUE THREAD

**Issue by mohamede1945** [quran/quran-ios#101]
**From collion,**
The pie charts on the juz screen made me think it was actually my reading progress.

--
I don't know what should be the best solution for this.

**Comment by thelittlehawk** [comment]
Is this still open/active?

**Comment by mohamede1945** [comment]
Yes and I still don't know the best solution for that. Do you have recommendations?
